pymedphys/cli/__init__.py

```diff
@@ -70,7 +67,18 @@ def pymedphys_cli():
     parser = define_parser()
 
     args, remaining = parser.parse_known_args()
-    logging.basicConfig(level=args.loglevel)
+
+    loglevel = logging.WARNING
+    logformat = "%(asctime)s %(levelname)-8s %(message)s"
+
+    if args.logging_verbose:
+        loglevel = logging.INFO
+
+    if args.logging_debug:
+        loglevel = logging.DEBUG
+        logformat += "\n    %(pathname)s#%(lineno)d"
```

Comment:
So, if `--debug` is passed to the CLI I have made the logging also include the file's full path as well as the line number of the log message.

When in not in debug I haven't included this as I think it will likely only be helpful when in debug mode.